### ISSUE TITLE
Harness Connector Resource Returns Error

### ISSUE THREAD

**Issue by jrmash** [harness-io/terraform-provider-harness#57]
Creating new `harness_connector` resources at the account level resulted in a `404 Not Found` error. Importing an existing connector at the account level produced a `Cannot import non-existent remote object` error.

The account id and API keys are set via environment variables, and I can successfully create and import other NG resources (`harness_organization`, for instance).

### Terraform Version
```
❯ terraform --version
Terraform v1.1.3
on darwin_amd64

❯ terraform providers
Providers required by configuration:
.
└── provider[registry.terraform.io/harness-io/harness] ~> 0.1.7
```

### Affected Resource(s)
- [harness_connector](https://registry.terraform.io/providers/harness-io/harness/latest/docs/resources/connector)
  - [gcp](https://registry.terraform.io/providers/harness-io/harness/latest/docs/resources/connector#nestedblock--gcp)
  - [git](https://registry.terraform.io/providers/harness-io/harness/latest/docs/resources/connector#nestedblock--git)
  - [github](https://registry.terraform.io/providers/harness-io/harness/latest/docs/resources/connector#nestedblock--github)

### Terraform Configuration Files
- [config file](https://gist.github.com/jrmash/217e491fa4d7f13452c4c1aee29634b8)

### Debug Output
- [apply output](https://gist.github.com/jrmash/4afe839e73cabf84f7000dcc67dd772f)
- [import output](https://gist.github.com/jrmash/58403216fac1bc2e882b85638803a5c6)

### Expected Behavior
The `apply` command should have created the resource in Harness NG at the account level.
The `import` command should have imported the resource in Harness NG at the account level.

### Actual Behavior
The `terraform apply` command produced a `404 Not Found` error
The `terraform import` command produced a `Cannot import non-existent remote object` error

### Steps to Reproduce
- `terraform apply`
- Manually create the resource in Harness NG
- `terraform import 'harness_connector.example' example_connector_f842600c`

### Important Factoids
- Environment variables are used to configure the provider
  - `harness_organization` can be used successfully (both create and import)
- The same behavior was observed with the `git` and `github` connector types, though I've only included config/output for the `gcp` type. Let me know if you need it for the others as well.

**Comment by micahlmartin** [comment]
@jrmash can you try version 0.1.9 and see you're still facing the issue?

**Comment by jrmash** [comment]
The issue persists, both in applying a new connector as well as importing an existing one.

**Comment by micahlmartin** [comment]
@jrmash would you mind trying the latest version and adding the TF_LOG=DEBUG option and emailing me the output? This will give me the exact output of the API calls to help me troubleshoot it. micah.martin@harness.io. Thanks!

**Comment by jrmash** [comment]
It looks like some changes made in `0.1.10` resolved the issue, and the added logging in `0.1.12-beta1` identified a misconfiguration that was masking the fix.

Thanks, @micahlmartin!
